internal/interconnections/create.go

```diff
@@ -2,15 +2,18 @@ package interconnections
 
 import (
 	"context"
+	"errors"
 	"fmt"
 
 	metal "github.com/equinix/equinix-sdk-go/services/metalv1"
 	"github.com/spf13/cobra"
 )
 
 func (c *Client) Create() *cobra.Command {
-	var name, metro, redundancy, connType, projectID, organizationID string
+	var name, metro, redundancy, connType, projectID, organizationID, svcTokenType string
```

Comment:
We should add e2e or mock tests covering this behavior (this PR may be why you asked about mocking yesterday; I'm just mentioning tests here so it's visible to other reviewers).

Reply:
There were not tests added for any operation interconnections subcommand yet. Not aware about the reason behind it. Adding tests just for new flag would anyway required tests

The mock package I was asking was for packngo to metal-go changes in CPEM as used [currently here](https://github.com/kubernetes-sigs/cloud-provider-equinix-metal/blob/0ac2e0f8959d3f8d7592b6c46651974818f3e242/metal/devices_test.go#L10)

Reply:
maybe we can cover tests in a separate PR. As anyway the interconnections subcommand is not yet released to prod

Reply:
It's fine to introduce tests in a separate PR, but in that case the test PR should be merged first so we can use them to validate this change.

Reply:
ohkay, I'll work on tests then and park this PR

Reply:
@ctreatma I am thinking, instead of making this PR blocked, we can add tests only to test the new capabilities added in this PR and will create another github issue for tests to be added for older code

Reply:
yes, that's a good idea

Reply:
@ctreatma Due to this bug we discussed yesterday https://github.com/equinix/metal-cli/issues/430
It wont be able to test the create connections from cli as we donot have ID of connection to fetch from server and verify in tests

Should we continue with the PR and wait for the schema level fixes to go in first and then pickup the tests. Anyway these are the new features and not officially released yet